Android/src/org/droidplanner/android/fragments/ParamsFragment.java

```diff
@@ -175,6 +175,12 @@ public boolean onQueryTextChange(String s) {
                 return true;
             }
         });
+        searchParams.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                searchParams.findViewById(android.support.v7.appcompat.R.id.search_button).callOnClick();
+            }
```

Comment:
@TShapinsky nice find! Store the retrieved view within an instance field instead. it should be available as soon as `onViewCreated` is called.